src/flask_session/sessions.py

```diff
@@ -498,12 +632,26 @@ def __init__(self, session_id, data, expiry):
                 self.expiry = expiry
 
             def __repr__(self):
-                return '<Session data %s>' % self.data
+                return f"<Session data {self.data}>"
+
+        from sqlalchemy import inspect
+
+        with app.app_context():
+            if not inspect(db.engine).has_table("Session"):
+                self.db.create_all()
```

Comment:
Hmm why to call `create_all` here? This will affect all tables instead of just the needed one, which will be very confusing

Reply:
Something like this should not be there at all! Any developer who uses alembic to maintain DB creation etc will be VERY confused and eventually pissed when finding out that some library executes DDL on its own.